glib/src/boxed_inline.rs

```diff
@@ -50,6 +51,7 @@ macro_rules! glib_boxed_inline_wrapper {
 
         impl $(<$($generic $(: $bound $(+ $bound2)*)?),+>)? std::clone::Clone for $name $(<$($generic),+>)? {
             #[inline]
+            #[allow(unknown_lints, clippy::incorrect_clone_impl_on_copy_type)]
```

Comment:
Why do we get this clippy lint at all? This should never be a `Copy` type. `BoxedInline` might, but all the others not.

Something seems fishy here :)

Reply:
It should be only needed for BoxedInline right.

Reply:
Didn't investigate further but see https://github.com/gtk-rs/gtk4-rs/actions/runs/5557902405/jobs/10152309339?pr=1423

Reply:
That seems valid to ignore here, yes.